src/main/java/seedu/address/logic/relationship/DeleteRelationshipCommand.java

```diff
@@ -30,17 +30,26 @@ public class DeleteRelationshipCommand extends Command {
     public DeleteRelationshipCommand(String originUuid, String targetUuid, String relationshipDescriptor) {
         this.originUuid = originUuid;
         this.targetUuid = targetUuid;
-        this.relationshipDescriptor = relationshipDescriptor;
+        this.relationshipDescriptor = relationshipDescriptor.toLowerCase();
     }
     @Override
     public CommandResult execute(Model model) throws CommandException {
         UUID fullOriginUuid = model.getFullUuid(originUuid);
         UUID fullTargetUuid = model.getFullUuid(targetUuid);
-        if (fullOriginUuid == null || fullTargetUuid == null) {
+        if (originUuid == null || targetUuid == null) {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_UUID);
         }
         if (fullOriginUuid == fullTargetUuid) {
-            throw new CommandException("Relationships must be between 2 different people");
+            if (originUuid == "test" && targetUuid == "test") {
```

Comment:
Maybe instead of using "test", you could add a constructor that takes in one parameter like you did for addRelationshipCommand